Makefile

```diff
@@ -289,9 +289,9 @@ govet: get-ci-tools
 	GOWORK=off $(CI_TOOLS_REPO_DIR)/test-runner/govet.sh ./api
 
 # Run go test against code
-gotest: get-ci-tools
-	GOWORK=off $(CI_TOOLS_REPO_DIR)/test-runner/gotest.sh
-	GOWORK=off $(CI_TOOLS_REPO_DIR)/test-runner/gotest.sh ./api
+gotest: envtest get-ci-tools
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" GOWORK=off $(CI_TOOLS_REPO_DIR)/test-runner/gotest.sh
```

Comment:
I'm not a fan of the scripts in the CI_TOOLS_REPO as I have not much control over it so I try to avoid them. In nova we ended up basically making gotest an alias for the test target. And having the test target run every tests. If we ever need both functional envtest and simple unit test then we might need to split that target to two but so far that worked OK for us.